test/test_support.ts

```diff
@@ -253,13 +253,12 @@ export function extractInlineSourceMap(source: string): BasicSourceMapConsumer {
 }
 
 export function findFileContentsByName(filename: string, files: Map<string, string>): string {
-  const filePaths = toArray(files.keys());
-  for (const filepath of filePaths) {
+  for (const filepath of files.keys()) {
     if (path.parse(filepath).base === path.parse(filename).base) {
       return files.get(filepath)!;
     }
   }
-  assert(undefined, `Couldn't find file ${filename} in files: ${JSON.stringify(filePaths)}`);
```

Comment:
I think the reason for the JSON.stringify here is that it quotes the elements of the list.  I think we want to keep that.

files.toString()
"a,b.ts"
JSON.stringify(files)
"["a","b.ts"]"

Reply:
Done.